backend/src/nodes/ncnn_nodes.py

```diff
@@ -138,13 +139,19 @@ class NcnnUpscaleImageNode(NodeBase):
     def __init__(self):
         super().__init__()
         self.description = "Upscale an image with NCNN. Unlike PyTorch, NCNN has GPU support on all devices, assuming your drivers support Vulkan."
-        self.inputs = [NcnnNetInput(), ImageInput()]
+        self.inputs = [
+            NcnnNetInput(),
+            ImageInput(),
+            NumberInput("Tile Size Target", default=0, minimum=0, maximum=None),
+        ]
         self.outputs = [ImageOutput()]
         self.category = NCNN
         self.name = "Upscale Image"
         self.icon = "NCNN"
         self.sub = "Processing"
 
+        self.split_factor = None
```

Comment:
I don't like that you added mutable state to the node. I can already see this causing bugs. Couldn't you give the `upscale` another parameter instead?

Reply:
Yeah